optimizely/helpers/validator.py

```diff
@@ -105,7 +105,14 @@ def are_attributes_valid(attributes):
     Boolean depending upon whether attributes are in valid format or not.
   """
 
-  return type(attributes) is dict
+  if type(attributes) is not dict:
+    return False
+
+  for attr_key in attributes.keys():
+    if not isinstance(attr_key, string_types):
```

Comment:
same as above to check string type.

Reply:
This is needed to write common code for both python versions 2 and 3.